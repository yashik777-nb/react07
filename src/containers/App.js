import "./App.css";
import AllIssuesPage from "./AllIssues/AllIssuesPage";

function App() {
  return (
    <div className="App">
      <AllIssuesPage />
    </div>
  );
}

export default App;
