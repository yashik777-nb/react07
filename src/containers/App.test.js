import { configure, mount } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import App from "./App";

configure({ adapter: new Adapter() });

describe("App page tests", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<App />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  test("App Snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
