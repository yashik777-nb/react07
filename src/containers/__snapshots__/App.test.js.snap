// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App page tests App Snapshot 1`] = `ReactWrapper {}`;
